avocado/core/safeloader.py

```diff
@@ -47,6 +47,41 @@ def __init__(self, path):
         with open(self.path) as source_file:
             self.mod = ast.parse(source_file.read(), self.path)
 
+    def is_avocado_test(self, klass):
```

Comment:
I do like this way better than the original location. Still the same issue as previous commit, but again, won't break anything so I'm fine...